FrontMen/fm-hours
# Feat/number pad

# Changes

- Add number pad for mobile
- Add 24h format input
## Related issues
#152 


## Added



## Removed



## Changed



## How to test
Easiest way to test on your mobile: 

Check out the Netlify deploy preview link. 
If you need to log in, add the preview link to authorised domains in the production firestore (don't forget to remove it after testing!)
- test if your phone shows a number pad on input
- check if values are correct



## Screenshots
![Screenshot 2021-07-16 at 14 03 55](https://user-images.githubusercontent.com/37107741/125944656-a57a5a00-de31-48d9-9539-f904ae2a2fd0.png)

![Screenshot 2021-07-12 at 16 33 29](https://user-images.githubusercontent.com/37107741/125305869-031afb00-e32f-11eb-8f0f-26d9f869ced6.png)
